mypy.ini

```diff
@@ -1712,6 +1712,16 @@ disallow_untyped_defs = true
 warn_return_any = true
 warn_unreachable = true
 
+[mypy-homeassistant.components.linear_garage_door.*]
```

Comment:
Can you explain why you need all of these?

Reply:
It's part of strict typing